index.js

```diff
@@ -124,7 +602,7 @@ app.intent('send.message - yes', async conv => {
   await circuit.addTextItem(convId, message);
   conv.contexts.delete('sendmessage_data');
   conv.ask('Message sent. Is there anything else I can do for you?');
-  conv.ask(new Suggestions('No, that\'s all', 'Yes'));
+  conv.ask(new Suggestions("No, that's all", 'Yes'));
```

Comment:
Our coding guidelines require single quotes for static text, or backticks if there is dynamic content or single quotes in the text.

You can use: `No, that's all`

Reply:
Reverted change.